### ISSUE TITLE
can't choose screen to apply

### ISSUE THREAD

**Issue by rhetr** [MiJyn/openlux#9]
when I have two screens connected to my computer I'm unable to choose which screen to apply openlux to. Is this already possible via terminal emulator and i'm just unaware of that feature or is this something that needs to be addressed in the code?

**Comment by AnonymousMeerkat** [comment]
Indeed, this needs to be addressed. I currently don't have any means of testing this, but I'll see what I can do :)
